admob_mediation/src/main/java/com/hmscl/huawei/admob_mediation/all_ads.kt

```diff
@@ -100,9 +146,9 @@ class all_ads : Adapter(),
         mediationExtras: Bundle?
     ) {
         try {
-            Log.d(TAG,"enter requestInterstitialAd")
-            if (serverParameters.isNullOrEmpty()){
-                Log.d(TAG,"Interstitial serverParameter is empty or null")
+            Log.d(TAG, "enter requestInterstitialAd")
+            if (serverParameters.isNullOrEmpty()) {
```

Comment:
you should enable only when build config is debug(you can check with this code BuildConfig.DEBUG)